### ISSUE TITLE
508-defect-2 [AXE, SCREENREADER]: Wizard, link's aria-describedby ID does not exist on the page

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#22729]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## Details

The green CTA link (styled as a button) has an `aria-describedby` that's pointing to an ID that doesn't exist as caught by AXE. I'm unsure what the ID was meant to be linked to. My best guess is probably the information below or above that CTA.

## Acceptance Criteria

- [ ] AXE no longer throws an error regarding a missing element ID

## Steps to Recreate

Run axe-coconut on a scenario where the green CTA is present

## Proposed Solution

Depends on design intent

## Screenshots or Trace Logs

![image](https://user-images.githubusercontent.com/14154792/113759200-cfeb1f00-96e2-11eb-8e65-5d7d8ee0ffff.png)

**Comment by denisecoveyduc** [comment]
@joshkimux @AngelaFowler82  I am doing a little GH clean up and wondered if you could just take a quick look at this issue for me to validate that this is still a concern? Since (and before I joined the team) the FSR has gone through many changes and this may or may not have already been addressed. Thanks in advance for checking it out for us.

**Comment by AngelaFowler82** [comment]
Oh the wizard, the not-so-wonderful wizard. The wizard is being retired, and a new and better component will soon take its place. I question the wisdom of retrofitting this only to replace it later. Can we instead make it a priority to implement the new component as soon as it becomes available?

**Comment by joshkimux** [comment]
Agree with Angela's assessment here. The [design system team has confirmed that the dynamic wizard will be deprecated](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/399#issuecomment-990011476), so I'm excited to see how this can be improved in the future. Using one thing per page to provide feedback to Veterans on what actions to take (as opposed to listing all questions dynamically on one page) will be significantly more accessible to Veterans with screen readers and/or cognitive disabilities. Excited to work with the team on transitioning this to something more accessible once the time comes 🥳 

<img width="964" alt="Screen Shot 2022-01-12 at 10 33 22 AM" src="https://user-images.githubusercontent.com/14154792/149217457-36b38212-fbd1-4e89-904d-7dbf63e579df.png">

That being said, @denisecoveyduc we can still close out this ticket as this is no longer an issue (the `aria-describedby` was removed, and an aria-live region successfully announces out the new section) 😄
